### PR TITLE
fixes openshift/openshift-docs#11983

### PR DESCRIPTION
**Pull request by davivcgarcia**
This pull-request implements the suggestion of #11983.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@davivcgarcia, does this apply to versions before 3.10?

File: upgrading/os_upgrades.adoc [hunk 1]
```diff
@@ -31,11 +31,19 @@ Use the following to safely upgrade the OS on a host:
 $ oc adm drain <node_name> --force --delete-local-data --ignore-daemonsets
 ----
 
-. Update the host packages, and reboot the host. A reboot ensures that the host is
+. Make sure all the sensitive packages are protected by exclude rules, update
```
Comment (kalexand-rh) [inline upgrading/os_upgrades.adoc]: Please add a separate step to enable the excluder rules. I'd restore line 34 to its original state and keep the update and reboot commands in this step.

In the new step, please give a bit more context about why you'd want to do enable these rules.

**Review by kalexand-rh — CHANGES_REQUESTED** [review]
@davivcgarcia, I think you should add a step instead of adding more complexity to this one.

**Review by kalexand-rh — APPROVED** [review]
Thanks!

**Comment by kalexand-rh** [comment]
@davivcgarcia, will you please squash your commits?

@jianlinliu, will you PTAL?

**Comment by davivcgarcia** [comment]
@kalexand-rh sorry this mess, I'm still kinda newbie around here. Please let me know if the PR looks good now!

**Comment by davivcgarcia** [comment]
@kalexand-rh answering your previous question, this update should be applicable for all versions.

**Comment by kalexand-rh** [comment]
No worries, @davivcgarcia. It looks much better now. :) 

@jianlinliu, do you agree that we should apply this change going back to 3.6?

**Comment by jianlinliu** [comment]
Yeah, agree to apply this change going back to 3.6

**Pull request merged**